tests/fast-integration/gardener/client.js

```diff
@@ -38,14 +45,69 @@ class GardenerClient {
 
     this.coreV1API = kc.makeApiClient(k8s.CoreV1Api);
     this.dynamicAPI = kc.makeApiClient(k8s.KubernetesObjectApi);
+    this.watch = new k8s.Watch(kc);
+    this.shootTemplate = config.shootTemplate;
+  }
+
+  async createShoot(shootName) {
+    debug(`Creating a K8S cluster in gardener namespace`);
+    if (!this.shootTemplate) {
+      debug(`No shoot Template`);
+      return new Error(`no shoot template defined in the Gardener client`);
+    }
+
+    const data = fromBase64(this.shootTemplate);
+
+    const replaced = data.replace(/<SHOOT>/g, shootName);
+
+    await this.dynamicAPI.create(JSON.parse(replaced))
+        .catch((err) => {
+          const response = JSON.stringify(err);
+          debug(`Got the error with response ${response}`);
+        });
+
+    await this.waitForShoot(shootName);
+
```

Comment:
nit: I personally prefer fewer empty lines :)

Reply:
I would like to leave empty lines as the are to increase readability. Additionally linter does not seem to complain about it so I think we are good.